client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -138,21 +140,26 @@ class EmailProvidersComparison extends Component {
 		this.isMounted = false;
 	}
 
-	getDefaultExpandedState( source ) {
-		if ( source === 'google-sale' ) {
+	getDefaultExpandedState( shouldPromoteGoogleWorkspace ) {
+		if ( shouldPromoteGoogleWorkspace ) {
 			return {
 				forwarding: false,
 				google: true,
 				titan: false,
 			};
 		}
+
 		return {
 			forwarding: false,
 			google: false,
 			titan: true,
 		};
 	}
 
+	shouldPromoteGoogleWorkspace( { gSuiteProduct, isGSuiteSupported, source } ) {
```

Comment:
This method is unused. Looks like you wanted to use it in the `connect` method?

Reply:
I've removed it.